app/grandchallenge/evaluation/forms.py

```diff
@@ -148,6 +148,15 @@ def clean(self):
                 "the submissions open date."
             )
 
+    def clean_submission_limit(self):
+        submission_limit = self.cleaned_data["submission_limit"]
+        if submission_limit > 0 and not self.instance.method_set.last().ready:
```

Comment:
There is a shortcut for this already, it should be used instead:

```suggestion
        if submission_limit > 0 and not self.instance.latest_ready_method:
```

Reply:
Rest looks great, thanks!

Reply:
The `latest_ready_method` is defined on the `Submission` model, not the `Phase` model. Should I add a similar method there?

Reply:
I just went ahead and added this property to the `Phase` model ;-)

Reply:
Hmm ok, now we have two ways of doing this, we should only have one. Indeed this should be a property of `Phase`. Could you unify the implementations, and add explicit ordering for the queryset so that `last` always returns the correct value?